Gems/ScriptCanvas/Code/Include/ScriptCanvas/Debugger/ClientTransceiver.cpp

```diff
@@ -168,7 +168,7 @@ namespace ScriptCanvas
         AzFramework::RemoteToolsEndpointInfo ClientTransceiver::GetNetworkTarget()
         {
             AzFramework::RemoteToolsEndpointInfo targetInfo;
-            if (AzFramework::IRemoteTools* remoteTools = RemoteToolsInterface::Get())
+            if (RemoteToolsInterface::Get())
             {
                 targetInfo = RemoteToolsInterface::Get()->GetDesiredEndpoint(ScriptCanvas::RemoteToolsKey);
             }
```

Comment:
minor: Maybe use the `remoteTools` instead to avoid calling get twice?

````
if (AzFramework::IRemoteTools* remoteTools = RemoteToolsInterface::Get();
    remoteTools != nullptr)
{
    targetInfo = remoteTools->GetDesiredEndpoint(ScriptCanvas::RemoteToolsKey);
}
````

Reply:
Sure thing. (remoteTools != nullptr is not needed though)